nncf/config/schema.py

```diff
@@ -713,19 +713,34 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False
 }
 
+KNOWLEDGE_DISTILLATION_ALGO_NAME_IN_CONFIG = 'knowledge_distillation'
+KNOWLEDGE_DISTILLATION_SCHEMA = {
+    **BASIC_COMPRESSION_ALGO_SCHEMA,
+    "properties": {
+        "algorithm": {
+            "const": KNOWLEDGE_DISTILLATION_ALGO_NAME_IN_CONFIG
+        },
+        "type": with_attributes(_STRING, description="Type of KDLoss (mse/softmax)")
```

Comment:
please avoid contractions in the documentation. Names like `KDLoss` is ok for the code, but not for the users reading documentation. Please go through all added docstrings and fix it.